Accounts/sharestatewithaccount/build.gradle

```diff
@@ -39,13 +49,14 @@ buildscript {
 }
 
 allprojects {
-    apply plugin: 'kotlin'
+    apply plugin: 'org.jetbrains.kotlin.jvm'
 
     repositories {
         mavenLocal()
 
         mavenCentral()
         maven { url 'https://download.corda.net/maven/corda-dependencies' }
+        maven { url 'https://download.corda.net/maven/corda-releases' }
```

Comment:
Wondering what this was required for? Don't think we should be needing to add additional repo's.

Reply:
Looks like it's needed for corda-core? That's what I'm getting with `corda-releases`removed:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':contracts:compileKotlin'.
> Could not resolve all files for configuration ':contracts:compileClasspath'.
   > Could not find net.corda:corda-core:4.12-RC01.
     Searched in the following locations:
       - file:/Users/jakub.zadroga/.m2/repository/net/corda/corda-core/4.12-RC01/corda-core-4.12-RC01.pom
       - https://repo.maven.apache.org/maven2/net/corda/corda-core/4.12-RC01/corda-core-4.12-RC01.pom
       - https://download.corda.net/maven/corda-dependencies/net/corda/corda-core/4.12-RC01/corda-core-4.12-RC01.pom
       - https://jitpack.io/net/corda/corda-core/4.12-RC01/corda-core-4.12-RC01.pom
       - https://download.corda.net/maven/corda-lib/net/corda/corda-core/4.12-RC01/corda-core-4.12-RC01.pom
       - https://repo.gradle.org/gradle/libs-releases/net/corda/corda-core/4.12-RC01/corda-core-4.12-RC01.pom
     Required by:
         project :contracts
```